lms/djangoapps/courseware/grades.py

```diff
@@ -499,10 +559,13 @@ def _progress_summary(student, course, course_structure=None):
         course_structure = get_course_blocks(student, course.location)
     if not len(course_structure):
         return None
-    scorable_locations = [block_key for block_key in course_structure if possibly_scored(block_key)]
-
-    with outer_atomic():
-        scores_client = ScoresClient.create_for_locations(course.id, student.id, scorable_locations)
+    if scores_client is None:
+        if grading_context_result is None:
+            scorable_locations = [block_key for block_key in course_structure if possibly_scored(block_key)]
+        else:  # TODO: these two list are equivalent, yeah?
```

Comment:
@ormsbee I may be misinterpreting the code.. the code that currently gets a subset must not want to present that data in the UI.  The UI simply loops through all returned blocks - and it only wants blocks that are graded, not just "possibly_scored".